run_route_scripts/run_with_server.py

```diff
@@ -22,33 +22,36 @@ def get_post_bodies(filename):
       yield post_body
 
 # for persistant connections
-def initialize(url_, output_dir_):
+def initialize(args):
```

Comment:
same changes made in the isodistances pr. just to streamline this implementation a bit